.github/workflows/links.yml

```diff
@@ -14,11 +14,16 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
+      - name: Remove symlink
+        run: rm docs/source/README.md
       - name: lychee Link Checker
         id: lychee
         uses: lycheeverse/lychee-action@v1.5.0
         with:
-          args: --accept=200,403,429  "**/*.html" "**/*.md" "**/*.txt" "**/*.json" --exclude "file:///github/workspace/*" --exclude-mail
+          args: --accept=200,403,429 "**/*.html" "**/*.md" "**/*.txt" "**/*.json" --exclude "file:///github/workspace/*" --exclude "file:///home/runner/work/opensearch-py/opensearch-py/docs/source/README.md" --exclude-mail 
```

Comment:
1. Can this be a relatie path, `docs/source/README.md`?
2. Lychee supports storing the list of files to exclude in a file, e.g. `.lychee.excludes`. Then you can use `--exclude-file .lychee.excludes` which makes this command line shorter. Let's use that.